packages/frontend-shared/src/components/ShikiHighlight.vue

```diff
@@ -107,8 +107,8 @@ export { highlighter, inheritAttrs }
 <script lang="ts" setup>
 import type { Ref } from 'vue'
 import { computed, onBeforeMount, ref } from 'vue'
-import { useClipboard } from '@vueuse/core'
-import CopyButton from './CopyButton.vue'
+import CopyButton from '../gql-components/CopyButton.vue'
```

Comment:
Need to remove the `isSupported` references in the template.

Reply:
Done in 7ca5cc5